daq/faucet_state_collector.py

```diff
@@ -84,7 +90,25 @@ def get_switches(self):
             switch_data[switch_name] = self.get_switch(switch_name)
         return switch_data
 
-    def get_switch_map(self):
+    def __get_egress_state(self):
+        """Return egress state obj"""
```

Comment:
Temporary placeholder format.

Reply:
If it's temporary then indicate so as a code comment -- the comment in the review is very transitory.